addons/project/static/src/components/project_right_side_panel/project_right_side_panel.scss

```diff
@@ -211,13 +79,25 @@ $o-rightpanel-filter-default-color: #D59244;
                         flex-basis: 33.33%;
 
                         &:nth-child(3n):not(:last-child) {
-                            border-right-width: 0px;
+                            border-right-width: 0px !important;
+                        }
+
+                        &:nth-child(4n):not(:last-child){
+                            border-right-width: 1px;
                         }
                     }
                 }
+            }
+        }
 
+        .o_rightpanel_header {
+            @include media-breakpoint-down(md) {
+                cursor: pointer;
```

Comment:
cursor on mobile? Not sure to understand the issue.

Reply:
On smaller screens there are these accordions that appear : https://www.awesomescreenshot.com/image/33101197?key=1db2ee38efb8c2537c1e3f9080c34670

It's weird if there is no cursor, it's mostly for small desktop windows